$OpenBSD: patch-t_t7501-commit_sh,v 1.1 2007/12/17 20:00:05 bernd Exp $

Backport git commit 41ec097aea91b4be038b1c4bbbe1bcb9d4e7aa53 to remove a
GNU sed(1) only option.

--- t/t7501-commit.sh.orig	Sat Dec  1 22:21:12 2007
+++ t/t7501-commit.sh	Sat Dec 15 00:53:34 2007
@@ -79,7 +79,8 @@ test_expect_success \
 
 cat >editor <<\EOF
 #!/bin/sh
-sed -i -e "s/a file/an amend commit/g" $1
+sed -e "s/a file/an amend commit/g" < $1 > $1-
+mv $1- $1
 EOF
 chmod 755 editor
 
@@ -98,7 +99,8 @@ test_expect_success \
 
 cat >editor <<\EOF
 #!/bin/sh
-sed -i -e "s/amend/older/g" $1
+sed -e "s/amend/older/g"  < $1 > $1-
+mv $1- $1
 EOF
 chmod 755 editor
 
